apps/electron-app/src/main/services/password-import/browser-detector.service.ts

```
@@ -0,0 +1,136 @@
+      if (a.browser > b.browser) return 1;
+      return b.lastModified.getTime() - a.lastModified.getTime();
+    });
+  }
+
+  /**
+   * Get Chrome profiles specifically
+   */
+  static getChromeProfiles(): BrowserProfile[] {
+    return this.findBrowserProfiles().filter(p => p.browser === "chrome");
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)